adobe/spectrum-web-components
# feat: add badge component

## Description

Implements the [Badge](https://spectrum.adobe.com/page/badge/) component.

## Related issue(s)

https://github.com/adobe/spectrum-web-components/issues/1643

## Motivation and context

Expanding component coverage

## How has this been tested?

1. Go to [the preview storybook](https://hunterloftis-badge--spectrum-web-components.netlify.app/storybook/?path=/story/badge--default)
2. Check out the various examples of badge usage
3. Go to [the preview docs site](https://hunterloftis-badge--spectrum-web-components.netlify.app/components/badge/)
4. Check out the documentation

## Types of changes


## Checklist


## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
